FocusIntentExtension/Info.plist

```diff
@@ -11,13 +11,13 @@
 	<key>CFBundleIdentifier</key>
 	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
-	<string>6.0</string>
+	<string>7.0.1</string>
```

Comment:
This is wrong and should never have been merged in. https://stackoverflow.com/questions/7010474/what-does-infodictionary-version-signify-in-information-property-list-plist-f

Reply:
Also https://stackoverflow.com/questions/7010474/what-does-infodictionary-version-signify-in-information-property-list-plist-f for confirmation